### DISCUSSION TITLE
How to add favourites?

### DISCUSSION THREAD

**Discussion by cmue** [4gray/iptvnator#292] [Q&A] [answered]
I've poked around, and cannot figure out how to favourite a channel. I've added a playlist via xtream codes and am using the mpv player for playback. I've tried right-clicking on a channel and dragging it to the Favourites button. Nothing. I am likely missing something obvious. Thanks in advance!

**Comment by 4gray** [comment, marked as answer]
That's right, unfortunately there is no favourites list for xtream playlist yet.

**Comment by holmesb** [comment]
New version (0.16.0) just released. I can't figure how to favourite a channel still. Is this on the roadmap?

**Comment by KiPSOFT** [comment]
https://github.com/4gray/iptvnator/pull/457
